src/system/settings.ts

```diff
@@ -39,6 +44,16 @@ type SystemSettingsConfig = {
     [key in `${typeof SYSTEM_ID}.${typeof SETTINGS.SHEET_EXPAND_DESCRIPTION_DEFAULT}`]: boolean;
 } & {
     [key in `${typeof SYSTEM_ID}.${typeof SETTINGS.SHEET_SKILL_INCDEC_TOGGLE}`]: boolean;
+} & {
+    [key in `${typeof SYSTEM_ID}.${typeof SETTINGS.AUTOMATION_TOKEN_FLAGS_BARS}`]: boolean;
+} & {
+    [key in `${typeof SYSTEM_ID}.${typeof SETTINGS.AUTOMATION_TOKEN_FLAGS_SIGHT}`]: boolean;
+} & {
+    [key in `${typeof SYSTEM_ID}.${typeof SETTINGS.AUTOMATION_TOKEN_FLAGS_SIZE}`]: boolean;
+} & {
+    [key in `${typeof SYSTEM_ID}.${typeof SETTINGS.TOKEN_DEFAULT_BAR_1_VAL}`]: string;
```

Comment:
For improved typing, you can use a string literal type here (Resource can be imported from types/cosmere)
```suggestion
    [key in `${typeof SYSTEM_ID}.${typeof SETTINGS.TOKEN_DEFAULT_BAR_1_VAL}`]: `resources.${Resource}`;
```